models/yang/sonic/sonic-sflow.yang

```diff
@@ -28,9 +35,12 @@ module sonic-sflow {
                     }
 
                     leaf collector_ip {
+          		mandatory true;
```

Comment:
indentation is inconsistent throughout this file

Reply:
Done